src/Node/Expression/OperatorEscapeInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+ namespace Twig\Node\Expression;
+
+/**
+ * Interface implemented by n-ary operators for n > 1.
```

Comment:
This interface can be defined on 1-ary operators, but that would not make sense.